### PR TITLE
docs(techstack): fix static image url

### PR DESCRIPTION
**Pull request by kherP**
fix static image url,
commented coverage block as we deleted test results folder om build

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -36,10 +36,10 @@ jobs:
...
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-        env:
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
+      # - name: publish coverage
+      #   uses: coverallsapp/github-action@v1.0.1
+      #   with:
+      #     github-token: ${{ secrets.GITHUB_TOKEN }}
+      #   env:
+      #     COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
```
Comment (yousifalraheem) [inline .github/workflows/publish.yml:44]: michael scott upset

Comment (kherP) [reply .github/workflows/publish.yml:44]: The test result folders are deleted before this task runs, I just commented this out temporarily and will uncomment it after we decide will to put the test result folder

Comment (kherP) [reply .github/workflows/publish.yml:44]: I will update the coverage path. Sorry for commented it out

**Review by omarVodiaK — DISMISSED** [review]

**Pull request merged**